### ISSUE TITLE
 couldn't reproducing the good effect shown by huggingface

### ISSUE THREAD

**Issue by v-yunbin** [coqui-ai/TTS#3245]
### Describe the bug

Using local model and interface on cpu, but I couldn't reproduce the good effect shown by huggingface.
The synthesized audio has a fast speech speed and wrong pauses, which is very different from the audio synthesized by the interface on the hugging face.

### To Reproduce

config = XttsConfig()
config.load_json("./source/model_v2/config.json")
model = Xtts.init_from_config(config)
model.load_checkpoint(config, checkpoint_dir="./source/model_v2/", eval=True)
#model.cuda()
model.to("cpu")
temperature=0.1
repetition_penalty=2.0
def generate(clone_audio_path, text, language):
    gpt_cond_latent, speaker_embedding = model.get_conditioning_latents(audio_path=clone_audio_path, gpt_cond_len=30, max_ref_length=60)
    chunks = model.inference_stream(text,language,gpt_cond_latent,speaker_embedding,repetition_penalty=7.0, temperature=0.85)
    wav_chuncks = []
    t0 = time.time()
    for i, chunk in enumerate(chunks):
        if i == 0:
           print(f"Time to first chunck: {time.time() - t0}")
        print(f"Received chunk {i} of audio length {chunk.shape[-1]}")
        wav_chuncks.append(chunk)
    wav = torch.cat(wav_chuncks, dim=0)
    #output_audio = rename_tool.path("audio", "wav")
    output_audio = "output.wav"
    torchaudio.save(output_audio, wav.squeeze().unsqueeze(0).cpu(), 24000)
    return output_audio
ref_wav = "samples/female.wav"
text = "习近平会见日本首相岸田文雄 当地时间11月16日下午，国家主席习近平在美国旧金山会见日本首相岸田文雄。习近平说，很高兴同首相先生再次会面。"
generate(ref_wav,text, "zh-cn")

### Environment

```shell
TTS 0.20.5
Python 3.10.13
torch  2.1.1
```

**Comment by v-yunbin** [comment]
anyone met the same problem?

**Comment by erogol** [comment]
temperature=0.1 very low. Don;t play with the default config parameters if you are after good perf.
